docs-site/docs/developers/events-checkins/data-model.md

```
@@ -0,0 +1,49 @@
+# Data Model & Schema
+
+The Events and Check-ins features utilize several interconnected tables in Supabase to manage state, visibility, and participation.
+
+---
+
+## 📊 Database Tables
+
+### `events`
+| Field | Type | Description |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)